package.json

```diff
@@ -30,37 +34,27 @@
-    "preinstall": "npm run check:node",
-    "prebootstrap:install": "npm run check:node",
-    "bootstrap": "npm run bootstrap:install -- --ci",
-    "bootstrap:prod": "npm run bootstrap:install -- --ci -- --only=prod",
-    "bootstrap:install": "lerna bootstrap --force-local",
-    "clean": "lerna clean -y",
-    "check:node": "check-node-version --node 16 --npm 8",
-    "fix": "npm run fix:js && npm run fix:style",
+    "clean": "pnpm -r --parallel exec rm -rf node_modules && rm -rf node_modules",
+    "fix": "pnpm fix:js && pnpm fix:style",
     "fix:js": "eslint --fix --ext .ts,.tsx,.js,.jsx src tests/performance-tests/src useful-scripts",
     "fix:style": "stylelint --fix src/**/*.{scss,css}",
     "format": "prettier --write \"{src,tests/performance-tests/src,useful-scripts}/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
     "format:check": "prettier --list-different \"{src,tests/performance-tests/src,useful-scripts}/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
     "format:robot": "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml tests/robot-tests",
-    "format:robot:check": "npm run format:robot -- --check --diff",
+    "format:robot:check": "pnpm format:robot -- --check --diff",
     "format:python": "python3 -m pipenv run flake8 tests && python3 -m pipenv run black tests && python3 -m pipenv run isort tests",
     "format:python:check": "python3 -m pipenv run flake8 tests --diff && python3 -m pipenv run black tests --diff && python3 -m pipenv run isort tests --diff",
-    "lint": "npm run lint:js && npm run lint:style",
+    "lint": "pnpm lint:js && pnpm lint:style",
     "lint:js": "eslint --ext .ts,.tsx,.js,.jsx src tests/performance-tests/src useful-scripts",
     "lint:style": "stylelint src/**/*.{scss,css}",
-    "start:admin": "cd src/explore-education-statistics-admin && npm start",
+    "start": "node useful-scripts/run.js",
     "start:azure": "cd \"C:\\Program Files (x86)\\Microsoft SDKs\\Azure\\Storage Emulator\" && c:AzureStorageEmulator.exe start -inprocess",
-    "start:docker:db": "cd src && docker-compose up db",
-    "start:frontend": "cd src/explore-education-statistics-frontend && npm run start:local",
-    "start:processor": "cd src/GovUk.Education.ExploreEducationStatistics.Data.Processor && func start --build",
-    "test": "lerna run test",
-    "test:snapshot": "lerna run test:snapshot",
-    "test:ci": "lerna run test:ci",
-    "test:coverage": "lerna run test:coverage",
-    "tsc": "lerna run tsc",
-    "install:azure-emulator": "curl --output installer.msi -L \"https://go.microsoft.com/fwlink/?linkid=717179&clcid=0x409\" && installer.msi && rm installer.msi",
-    "install:azure-tools": "npm install -g azure-functions-core-tools@4 --unsafe-perm true"
+    "test": "pnpm -r --no-bail test",
+    "test:snapshot": "pnpm -r --no-bail test:snapshot",
+    "test:ci": "pnpm -r --no-bail test:ci",
+    "test:coverage": "pnpm -r --no-bail test:coverage",
+    "tsc": "pnpm -r --no-bail tsc",
```

Comment:
As discussed on Slack, this won't output well on CI and will make it really difficult to track down build errors. We may want to use `--append-only=reporter`, but only do this with specific commands like `test:ci`

Reply:
Sounds good, I've added this in addition to the `--aggregate-output` flag to the `test:ci` and `tsc` commands